src/lexer.coffee

```diff
@@ -1178,6 +1178,12 @@ CSX_IDENTIFIER = /// ^
   ( (?: (?!\s)[\.\-$\w\x7f-\uffff] )+ ) # Like `IDENTIFIER`, but includes `-`s and `.`s.
 ///
 
+# Fragment: <></>
+CSX_FRAGMENT_IDENTIFIER = /// ^
+  (?![\d<]) # Must not start with `<`.
+  ( (?=>) ) # Ends immediately with '>.
```

Comment:
why does this need to be a lookahead?

Reply:
For normal CSX tags RegEx will capture tag name, e,g, `div` from `<div>`.
The fragment tag is empty (`<>`) which means that tag name is '', and has to be compared to '' from closing tag `</>`.
I'm far from RexEg mastery and open to any better suggestion.

Reply:
Aren’t you just trying to match exactly `<>` or `</>`? So

```js
/(<>)|(<\/>)/
```
https://regex101.com/r/68Imt8/2

Reply:
Yes, something like that.  But, the way `CSX` processing works in the `lexer` is more complex and I'm trying to reuse existing code by adding RegEx for empty tag `<>`. This basically means I need to capture `` in `<>` which is then used for creating `CSX` token and finding the matching closing tag.
I'm working on RegEx improvement at the moment.